bar.hpp

```diff
@@ -0,0 +1,12 @@
+namespace test
+{
+
+class Bar
+{
+public:
+    Bar(int x);
+    void print();
+    int m_x;
```

Comment:
Make this field private (code style 4.6) [Code style: Access control](https://google.github.io/styleguide/cppguide.html#Access_Control)

Reply:
fixed